troubleshooting.md

```diff
@@ -46,6 +46,11 @@
     - Reinstall helm with the yaml set to FC
   - iSCSI stack on linux host is not using unique IQN or other open-iSCSI issue.
     - On fresh installations and clones of VM's I often must restart open-iscsi on the worker nodes.
+  - The Pure provisioner doesn't have the proper permissions to provision Kubernetes PV resources. 
```

Comment:
I think this should be worded differently.
Since giving the provisioner a cluster-admin role is equivalent to setting the setuid bit on the Unix process, we dont grant that rule automatically, on purpose.

Reply:
My suggestion :
If you run into this error message <insert error message here>, you have not granted cluster-admin role to the pure provisioner. You can grant that by running the command : <insert command here>

Reply:
I agree. I will finesse the wording some.

Reply:
@taherv I changed the wording.